src/components/Checkbox/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BaseCheckbox from './BaseCheckbox';
+
+const Checkbox = props => (
+    <BaseCheckbox
+        onMouseDown={props.onMouseDown ? props.onMouseDown : props.onPress}
+        onPress={undefined}
```

Comment:
```suggestion
```

Reply:
Actually setting the onPress undefined is intentional because it prevents it from executing twice on web

Reply:
But, I just realised a dumb mistake I made.. I am assigning it before spreading props so it has no effect anyway. So I guess we can remove it. I tested on chrome-arm64, safari and mWeb Chrome and seems to work fine without it.